resources/magento2/common.properties

```diff
@@ -27,3 +27,10 @@ common.notAvailable=N/A
 common.classInheritance=Inherit Class
 common.license.proprietary=Proprietary
 common.description=Description
+common.parentDirectory=Parent Directory
+common.cliCommandName=CLI Command Name
```

Comment:
What do you think if we'll keep it just `Command Name`?

Reply:
Agree, will change :)